crawlclima/redemet/tests/test_rmet.py

```diff
@@ -20,6 +22,13 @@
     redemet_url,
 )
 
+load_dotenv()
+
+'''
```

Comment:
I think you don't need this comment here ...

Reply:
Removed.